filters/filters-2020.txt

```diff
@@ -5694,6 +5694,12 @@ gamesrepacks.com##+js(nostif, mdpDeBlocker)
 ! https://forums.lanik.us/viewtopic.php?p=157292#p157292
 @@||firenzetoday.it^$ghide
 
+! SSAI Video ads on ABC Owned TV station sites https://github.com/uBlockOrigin/uBlock-issues/issues/760#issuecomment-715702997
+||content.uplynk.com/api/*&ad=$xhr,queryprune=|ad,domain=abc7ny.com|abc7.com|abc7chicago.com|6abc.com|abc7news.com|abc13.com|abc11.com|abc30.com
+
+! SSAI Video ads on Discovery TV sites https://github.com/uBlockOrigin/uBlock-issues/issues/760#issuecomment-715926907
+||api.discovery.com/v1/streaming/video/*&adNetworkId=$xhr,queryprune=|ad,domain=ahctv.com|animalplanet.com|cookingchanneltv.com|destinationamerica.com|discovery.com|discoverylife.com|diynetwork.com|foodnetwork.com|hgtv.com|investigationdiscovery.com|motortrend.com|sciencechannel.com|tlc.com|travelchannel.com
```

Comment:
@llacb47, in [this commit](https://github.com/AdguardTeam/AdguardFilters/commit/5e4fb7cdff04b1d608eb74751aaa9ab94519b59e) @AdamWr has replaced a proposed solution:
`@@||geolocation.onetrust.com/cookieconsentpub/v1/geo/location$script,domain=tvn24.pl` to
`@@||geolocation.onetrust.com/cookieconsentpub/*/geo/location$domain=tvn24.pl`

notice: `v1` ==> `*`, the same has not happened in the case of  the current commit:
`||api.discovery.com/v1/streaming/video/` is still `v1`

Now I wonder whether `v1` or `*` is the proper / safer / better solution.
Do you guys have just a different filters approach?
It would be nice to see a confrontation between both of you.
However anybody else can take a voice as well.

![](https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRWTnboCZCF_GSQipkteMS9hQ3CSfLv91JYug&usqp=CAU)